inc/cron_push.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Push cron function for Loco Translate module.
+ */
+
+/**
+ * Push assets & translations to Loco.
+ *
+ * @param string $langcode
+ *   The langcode to push from.
+ * @param array $settings
+ *   The configurations needed to push on Loco.
+ */
+function loco_translate_cron_push($langcode, array $settings) {
+  $loco_push = \Drupal::service('loco_translate.loco_api.push');
+
+  $loco_push->setApiClientFromConfig();
+  $path = $settings['template'];
+
```

Comment:
return and do nothing (notice?) if `$settings['template']` is not set.

Reply:
It's not possible to be empty - formsettings ensure it's not

Reply:
I still cover the case